hooks/src/index.js

```diff
@@ -220,12 +217,21 @@ if (typeof window !== 'undefined') {
 	};
 }
 
+function handleEffects(effects) {
+	effects.forEach(invokeCleanup);
+	effects.forEach(invokeEffect);
+	return [];
```

Comment:
Instead of returning maybe we could, just set the `effects = []`? Then we could call `handleEffects` directly with the `effects` list. That would shave `8B` from the current implementation.

Reply:
Actually, scratch that. These are not references.

Reply:
My first implementation did this aswell

Reply:
@cristianbote seems like the suggestion has been addressed. Can you unblock the PR?

Reply:
Sure thing 😊